README.md

```diff
@@ -1889,65 +1889,17 @@ the Supervisor on a device is outdated and is causing an issue. Usually the best
-v10.2.2: Pulling from balena/armv7hf-supervisor
-9876f0e6f2c9: Pull complete
-caa2e1d76000: Pull complete
-9d642e50fd65: Pull complete
-a575f4a66dd6: Pull complete
-b4d4863e8826: Pull complete
-9fb2abe4f9fb: Pull complete
-Total:  [==================================================>]  22.08MB/22.08MB
-Digest: sha256:12e63c0839a9c9fa00a29487271eb2efaf8716e4a8b4676aa195dd07d206b04f
-Status: Downloaded newer image for balena/armv7hf-supervisor:v10.2.2
-resin_supervisor
-Start supervisor...
-root@8117443:~#
-```
-
-This has moved the Supervisor from v9.2.2 to version v10.2.2, including
-downloading the required update and then installing it, and removing the old
-version of the Supervisor.
-
-The `-t` tag, as can be seen, specifies which version of the Supervisor to
-update to. Note that the Supervisor is built for several different
-architectures, and so a valid tag on the specific version is required to be
-built for this to work (in this case `armv7hf` as can be seen by the tag).
-
-There is an alternate way of updating the Supervisor, via the HUP scripts
-[here](https://github.com/balena-os/balenahup/). You should read the
-documentation before carrying out any sort of update, but in essence
-it's possible to carry out HUP scenarios that aren't possible via the
-standard `update-resin-supervisor` script. This includes updating
-development OS version. The following example, will remotely update
-the Raspberry Pi device with UUID `123457890abcdef` on the production
-balenaCloud environment to version `v2.43.0+rev1.dev` of balenaOS:
-
-```shell
-./upgrade-ssh-2.x.sh -s balenadevice --hostos-version 2.43.0+rev1.dev --assume-supported --ignore-sanity-checks --force-slug raspberrypi3 -u 123457890abcdef
-```
-
-Extreme care should be taken when carrying out updates like this, as
-they are not common and should only be used in extreme circumstances.
+You can write a script to manage this for a fleet of devices in combination with other SDK functions such as [device.getAll](https://www.balena.io/docs/reference/sdk/node-sdk/#devicegetalloptions-%E2%87%92-codepromisecode) or for single use instances you can quickly execute it by going to balenaCloud dashboard and open the developer tools console. In this enviornment the SDK is already available and contains your credentials so you just have to invoke the function.
+
+In the following snippet we are setting our device to run Supervisor `v12.1.1`. Be sure to change the `<uuid>` to the device you're trying to update. After this request is made the device will begin to download and run the target Supervisor version.
+
+```js
+sdk.models.device.setSupervisorRelease('<uuid>', 'v12.1.1').then(function() { console.log('DONE!') });
+```
+
+**Note:** You may recall using `update-resin-supervisor` for managing the Supervisor version on devices but we are no longer encouraging that you directly interact with it. Under the hood the device is still running this script though.
```

Comment:
```suggestion
**Note:** In order to use this method, you must have edit permissions on the device (i.e., more than just support access).
```

Reply:
Wow that was a big oversight on my part to not mention the permissions required.

Reply:
I guess we don't want to mention `update-resin-supervisor` at all anymore? I figured it might avoid some confusion of people saying "Wasn't there a script on device to do this?"

Reply:
I'd prefer to drop all mentions as i think they'll cause more confusion than they'll resolve.

Reply:
Sounds good to me. Removed it.